recipes/spice-utilities/meta.yaml

```diff
@@ -0,0 +1,368 @@
+    folder: utilities  # [osx]
+    sha256: c899d2a62afc781c317004c254b8c154409ade18e651654cfd2667ef2904b338  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/oem2spk  # [osx]
+    folder: utilities  # [osx]
+    sha256: fa5a5dfd2bee88f51ad5eb52af9d36f700c26d77a607b66a34ba4845b84c5c40  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/optiks  # [osx]
+    folder: utilities  # [osx]
+    sha256: 83aa6505079f28c7553a96bac88c12a6b4f8564a3da70eebd82851ecfa3aef8f  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/orbnum  # [osx]
+    folder: utilities  # [osx]
+    sha256: 293b88dfad418c0b908060796039a2e2e753dda1ef8c7e70326a1d4291bbb03f  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/pinpoint  # [osx]
+    folder: utilities  # [osx]
+    sha256: 802fdca8009fb394e45d32873339d71da218f5cdb2dc7411ca928c5651400007  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/prediCkt  # [osx]
+    folder: utilities  # [osx]
+    sha256: 53a6cd3b22a08666104087ed8bf11d3a249aefbd75e2771030855075c6e259fa  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/spacit  # [osx]
+    folder: utilities  # [osx]
+    sha256: ec36d06bf00bc088408652ea0b61dcb44734602d46c99a4fde894ecc06f5fe88  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/spk2oem  # [osx]
+    folder: utilities  # [osx]
+    sha256: f4a0af4633941159e11ef5961b78a1cf49883d089a85298e5375cfc2eb684d1e  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/spkdiff  # [osx]
+    folder: utilities  # [osx]
+    sha256: 4cf959e55117deadbcc7e36cd8a250a36506d129f8185931cc5ebdaf08389fb7  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/spkmerge  # [osx]
+    folder: utilities  # [osx]
+    sha256: fd903d40797d1f8d36e5bd78c4a403758421d328e8e8e9b110a660683604dad5  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/spy  # [osx]
+    folder: utilities  # [osx]
+    sha256: a6db357f4b8c85610f0835e8536650938943ce00c6b010462ecb8cfeb688fc30  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/starsb  # [osx]
+    folder: utilities  # [osx]
+    sha256: 27f22ab5b0da1b74cc5b6f8b955b8b2601b009e8c192c12760c9d12e3f50ce7a  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/tobin  # [osx]
+    folder: utilities  # [osx]
+    sha256: ecd64ceecacff5796f5108a63e265cbfe75345d7922d624cce8ced0b9786032a  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/toxfr  # [osx]
+    folder: utilities  # [osx]
+    sha256: a5c7f8ed96eaaaa87c3b53f833bdf3b85100b89825fdf87415673696aaf9a5ac  # [osx]
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - /lib64/libc.so.6  # [linux64]
+    - /lib64/libm.so.6  # [linux64]
```

Comment:
This is probably fine, but maybe someone with binary redistribution experience should weigh in: Should we instead be guaranteeing a compatible sysroot?

@conda-forge/staged-recipes

Reply:
I don't know enough about how conda works to know what is best here. I put these here to eliminate some errors in the build process I think?

Reply:
Typically you need permission from the @conda-forge/core team to redistribute pre-compiled binaries. I'm also not sure whether they want official record of correspondence with the JPL to prove that conda-forge has permission to redistribute. If you don't get a reponse from the core team. Please show up to one of the conda-forge community meetings.